data/al/legislature/Joe-Lovvorn-972dcfd0-f2a2-438b-b5de-276ceb398a91.yml

```diff
@@ -12,17 +12,23 @@ roles:
   district: '79'
 offices:
 - classification: capitol
-  address: 11 South Union Street, Suite 435, Montgomery AL
+  address: 11 South Union Street,Suite 435,Montgomery, AL 36130-2950
   voice: 334-261-0540
+- classification: district
+  address: '515 Ogletree Road,Office: ,Auburn, AL 36830'
```

Comment:
Looks like some of the offices were added to the API weird cuz it's also wrong on the source. Need to remove `Office: ,`